config/default/manager_webhook_patch.yaml

```diff
@@ -14,10 +14,10 @@ spec:
           protocol: TCP
         volumeMounts:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
-          name: cert
+          name: webhook-cert
           readOnly: true
       volumes:
-      - name: cert
+      - name: webhook-cert
         secret:
           defaultMode: 420
-          secretName: opendatahub-operator-controller-webhook-cert
+          secretName: opendatahub-operator-controller-manager-service-cert
```

Comment:
The OLM runtime consistently creates a volume named webhook-cert for serving the webhook server and does not use the volume name cert at all. To align with this, I updated the volume name in the config folder to webhook-cert, ensuring that make deploy also uses the same volume name for consistency across both deployment methods.

Similarly, for secretName, OLM creates the secret as <serviceName>-cert (e.g., opendatahub-operator-controller-manager-service-cert). I updated the config to use the same naming pattern, allowing the DSCInitialization reconcile logic to consistently locate the secret in both OLM and make deploy workflows, thereby injecting the caBundle into the webhook configuration.